src/GUI/UI/UIHouses.h

```diff
@@ -100,3 +114,6 @@ extern class GraphicsImage *_591428_endcap;
 extern std::array<const HouseAnimDescr, 196> pAnimatedRooms;
 
 extern IndexedArray<int, BUILDING_WEAPON_SHOP, BUILDING_DARK_GUILD> itemAmountInShop;
+
+extern std::vector<HouseInteractiveDesc> dialogueInteractiveList;
+extern int currentDialogueInteractive;
```

Comment:
If we rename the type to `HouseNpcDesc`, this turns into `houseNpcs` & `currentHouseNpc`. Would be nice to next move them somewhere and add a proper accessor, smth like `const HouseNpcDesc &currentHouseNpc()`.

Reply:
I called it "dialogue" because it's also used for conversation with overworld NPCs to store NPC data and face icon - functions that update NPC dialogue are the same for house and overworld NPCs.
But I understand what you mean my first take on the names was similar to this.